File: src/main/webapp/js/service/houseService.js
//服务层
app.service('houseService',function($http){
	var requestUrl  =  localStorage.getItem('requestUrl');
	//增加 
	this.add=function(entity){
		return  $http.post(requestUrl +  '/house/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post(requestUrl+ '/house/update',entity );
	}
	
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post(requestUrl+ '/house/list?page='+page+"&rows="+rows, searchEntity);
	} 
	
	//查询实体
	this.findOne=function(id){
		return $http.get(requestUrl +  '/house/findOne?id='+id);
	}
	
});

File: src/main/java/com/webproject/service/HouseService.java
package com.webproject.service;

import com.webproject.entity.House;
import com.webproject.util.PageResult;
import com.webproject.util.Result;

public interface HouseService {

	Result add(House house);

	PageResult list(House house, int page, int rows);

	House findOne(String id);

	Result update(House house);

}
